tests/integration/models/test_eval_methods_mass_balance.py

```diff
@@ -0,0 +1,420 @@
+                    0,
+                    0,
+                    0,
+                    0,
+                    1e6,
+                    0,
+                    1e6,
+                    0,
+                    1e6,
+                    0,
+                    1e6,
+                    0,
+                ]
+            ),
+            None,
+        ),
+        (
+            "darcy_flux",
+            np.array(
+                [
+                    -4.0e-13,
+                    -1.0e-12,
+                    4.0e-13,
+                    -4.0e-13,
+                    -1.0e-12,
+                    4.0e-13,
+                    -4.0e-13,
+                    -4.0e-13,
+                    -1.0e-12,
+                    -1.0e-12,
+                    4.0e-13,
+                    4.0e-13,
+                    1.0e-12,
+                    1.0e-12,
+                    1.0e-12,
+                    1.0e-12,
+                    -8.0e-15,
+                    -1.0e-12,
+                    8.0e-15,
+                    1.0e-12,
+                    -8.0e-15,
+                    -1.0e-12,
+                    8.0e-15,
+                    1.0e-12,
+                ]
+            ),
+            None,
+        ),
+        (
+            "equivalent_well_radius",
+            np.array([0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.2]),
+            None,
+        ),
+        ("fluid_compressibility", 4e-10, None),
+        ("fluid_density", 1000 * np.exp(4e-10 * 200 * pp.BAR), None),
+        (
+            "fluid_flux",
+            np.array(
+                [
+                    -4.00000000e-07,
+                    -1.00803209e-06,
+                    4.03212834e-07,
+                    -4.00000000e-07,
+                    -1.00803209e-06,
+                    4.03212834e-07,
+                    -4.00000000e-07,
+                    -4.00000000e-07,
+                    -1.00803209e-06,
+                    -1.00803209e-06,
+                    4.03212834e-07,
+                    4.03212834e-07,
+                    1.00803209e-06,
+                    1.00803209e-06,
+                    1.00803209e-06,
+                    1.00803209e-06,
+                    -8.00000000e-09,
+                    -1.00803209e-06,
+                    8.06425668e-09,
+                    1.00803209e-06,
+                    -8.00000000e-09,
+                    -1.00803209e-06,
+                    8.06425668e-09,
+                    1.00803209e-06,
+                ]
+            ),
+            None,
+        ),
+        (
+            "fluid_mass",
+            np.array(
+                [
+                    1.76405615e00,
+                    1.76405615e00,
+                    1.76405615e00,
+                    1.76405615e00,
+                    3.52811230e-02,
+                    3.52811230e-02,
+                    3.52811230e-02,
+                    3.52811230e-02,
+                    7.05622460e-04,
+                ]
+            ),
```

Comment:
For derived quantities s.a. mass, do you think it'd be useful to add a short comment explaining how it was computed? E.g. rho * phi * V

Reply:
Great. But it should not be an absolute requirement (e.g. not fluxes)